proposals/simd/BinarySIMD.md

```diff
@@ -30,210 +30,218 @@ In the description below, `ImmLaneIdx{I}` indicates the maximum value of the byt
+| `i16x8.gt_s`                |    `0x31`| -                        |
+| `i16x8.gt_u`                |    `0x32`| -                        |
+| `i16x8.le_s`                |    `0x33`| -                        |
+| `i16x8.le_u`                |    `0x34`| -                        |
+| `i16x8.ge_s`                |    `0x35`| -                        |
+| `i16x8.ge_u`                |    `0x36`| -                        |
+| `i32x4.eq`                  |    `0x37`| -                        |
+| `i32x4.ne`                  |    `0x38`| -                        |
+| `i32x4.lt_s`                |    `0x39`| -                        |
+| `i32x4.lt_u`                |    `0x3a`| -                        |
+| `i32x4.gt_s`                |    `0x3b`| -                        |
+| `i32x4.gt_u`                |    `0x3c`| -                        |
+| `i32x4.le_s`                |    `0x3d`| -                        |
+| `i32x4.le_u`                |    `0x3e`| -                        |
+| `i32x4.ge_s`                |    `0x3f`| -                        |
+| `i32x4.ge_u`                |    `0x40`| -                        |
+| `f32x4.eq`                  |    `0x41`| -                        |
+| `f32x4.ne`                  |    `0x42`| -                        |
+| `f32x4.lt`                  |    `0x43`| -                        |
+| `f32x4.gt`                  |    `0x44`| -                        |
+| `f32x4.le`                  |    `0x45`| -                        |
+| `f32x4.ge`                  |    `0x46`| -                        |
+| `f64x2.eq`                  |    `0x47`| -                        |
+| `f64x2.ne`                  |    `0x48`| -                        |
+| `f64x2.lt`                  |    `0x49`| -                        |
+| `f64x2.gt`                  |    `0x4a`| -                        |
+| `f64x2.le`                  |    `0x4b`| -                        |
+| `f64x2.ge`                  |    `0x4c`| -                        |
+| `v128.not`                  |    `0x4d`| -                        |
+| `v128.and`                  |    `0x4e`| -                        |
+| `v128.andnot`               |    `0x4f`| -                        |
+| `v128.or`                   |    `0x50`| -                        |
+| `v128.xor`                  |    `0x51`| -                        |
+| `v128.bitselect`            |    `0x52`| -                        |
+| `v128.load8_lane`           |    `0x58`| m:memarg, i:ImmLaneIdx16 |
+| `v128.load16_lane`          |    `0x59`| m:memarg, i:ImmLaneIdx16 |
+| `v128.load32_lane`          |    `0x5a`| m:memarg, i:ImmLaneIdx16 |
+| `v128.load64_lane`          |    `0x5b`| m:memarg, i:ImmLaneIdx16 |
+| `v128.store8_lane`          |    `0x5c`| m:memarg, i:ImmLaneIdx16 |
+| `v128.store16_lane`         |    `0x5d`| m:memarg, i:ImmLaneIdx16 |
+| `v128.store32_lane`         |    `0x5e`| m:memarg, i:ImmLaneIdx16 |
+| `v128.store64_lane`         |    `0x5f`| m:memarg, i:ImmLaneIdx16 |
```

Comment:
`ImmLaneIdx16` needs to be updated for {load,store}_{16,32,64}

Reply:
Fixed